tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "module": "system",
+    "target": "es6",
```

Comment:
We should NOT use es6 syntax. As of now, UI5 is a ES5 framework which supports also legacy browser IE11. With that ES6 can not be used out of the box. Without using transpiling (e.g. babel) the IE11 use case would not run.

Reply:
done